packages/mask/src/extension/popups/SSR-client.ts

```diff
@@ -0,0 +1,50 @@
+// This file should not import any file!!
+/// <reference types="web-ext-types" />
+
+// If the current page is "", navigate to "#/personas", therefore we can avoid a Router skip when hydrating.
+if (location.hash === '') location.assign('#/personas')
+
+declare const trustedTypes: any
```

Comment:
suggest, add comment for the variable. (why use `any` type)

Reply:
Trusted type API is not included in the `@types/web`. (It only shipped on Chrome). If we merge https://github.com/DimensionDev/Maskbook/pull/6207/files#diff-a8fdf1f6fd182cb22743da6888e64e555643bb8edd5f82772b023489f82ad32c, I can change its type to not any.